visualization/app/codeCharta/state/fileState.service.ts

```diff
@@ -33,6 +35,12 @@ export class FileStateService {
 		return this.fileStates
 	}
 
+	public isDeltaMode(): boolean {
```

Comment:
Actually there exist already a method called `FileStateHelper.isDeltaState()`. Is there a special reason to duplicate it? 😄